### PR TITLE
[BEAM-7630] add ITs for writing and reading bytes from pubsub

### PR DESCRIPTION
**Pull request by Juta**
Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_VR_Flink/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_VR_Spark/lastCompletedBuild/)
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Batch/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Streaming/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Spark_Batch/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python3_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) <br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python_PVR_Flink_Cron/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_VR_Spark/lastCompletedBuild/)

Pre-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

--- |Java | Python | Go | Website
--- | --- | --- | --- | ---
Non-portable | [Build Status](https://builds.apache.org/job/beam_PreCommit_Java_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Go_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Website_Cron/lastCompletedBuild/) 
Portable | --- | [Build Status](https://builds.apache.org/job/beam_PreCommit_Portable_Python_Cron/lastCompletedBuild/) | --- | ---

See [.test-infra/jenkins/README](https://github.com/apache/beam/blob/master/.test-infra/jenkins/README.md) for trigger phrase, status and link of all Jenkins jobs.

### CODE REVIEW

**Comment by Juta** [comment]
Run Python PostCommit

**Comment by Juta** [comment]
Run Python PostCommit

**Comment by Juta** [comment]
Run Python PostCommit

**Comment by Juta** [comment]
R: @tvalentyn

File: sdks/python/apache_beam/io/gcp/pubsub_integration_test.py [hunk 1]
```diff
@@ -94,6 +120,23 @@ class PubSubIntegrationTest(unittest.TestCase):
           }),
       ],
   }
+  EXPECTED_OUTPUT_MESSAGES_BYTES = {
+      'TestDirectRunner': [
+          PubsubMessage(b'data001\xab\xac-seen', {'processed': 'IT'}),
+          PubsubMessage(b'data002\xab\xac-seen', {
+              TIMESTAMP_ATTRIBUTE: '2018-07-11T02:02:50.149000Z',
```
Comment (tvalentyn) [inline sdks/python/apache_beam/io/gcp/pubsub_integration_test.py]: Thanks, let's combine this scenario with existing scenario.

**Comment by Juta** [comment]
Run Python PostCommit

**Comment by Juta** [comment]
Run Python PostCommit

File: sdks/python/apache_beam/io/gcp/pubsub_integration_test.py (hunk 1 above)
Comment (Juta) [reply sdks/python/apache_beam/io/gcp/pubsub_integration_test.py]: done

**Comment by tvalentyn** [comment]
run python postcommit

**Comment by tvalentyn** [comment]
The only test failure on this PR is `(apache_beam.io.gcp.big_query_query_to_table_it_test.BigQueryQueryToTableIT) ... FAIL`, which is a known flake unrelated to this change. 

@pabloem Could you help merging this PR please?

**Comment by tvalentyn** [comment]
The only test failure on this PR is (apache_beam.io.gcp.big_query_query_to_table_it_test.BigQueryQueryToTableIT) ... FAIL, which is a known flake unrelated to this change.

@pabloem Could you help merging this PR please?

**Comment by pabloem** [comment]
Okay fair enough. Thanks Valentyn!

**Pull request merged**